### ISSUE TITLE
Admin Panel Sidebar Redesign

### ISSUE THREAD

**Issue by cschieder** [kubermatic/dashboard#4988]
### Why? 
As more points get added the structure is becoming unclear

### Solution: 
Section the sidebar into overarching topics

![image](https://user-images.githubusercontent.com/49145827/190355794-ed6160e1-2b52-4450-9ee0-db65a5d41c01.png)

### Comparison Sceen:
![image](https://user-images.githubusercontent.com/49145827/190356174-5c2ef0a2-3857-4b68-b76f-6c628ff1b106.png)

### Implementation/Design Details
https://www.figma.com/file/dJQKgUPzIDND36o2Nq8dct/Loodse-UI?node-id=7679%3A81835
![image](https://user-images.githubusercontent.com/49145827/190356953-61d373a5-4816-4440-a6f6-15726715229f.png)

With the changing of the structure I propose moving this section of the "Interface" point
![image](https://user-images.githubusercontent.com/49145827/190411352-b538d141-b1e4-4c12-a07a-c75b9647d44d.png)

To Defaults and having Custom Links & the Display API Docs etc part of "Customization" 

### Minimized Version
https://www.figma.com/file/dJQKgUPzIDND36o2Nq8dct/Loodse-UI?node-id=7758%3A98874
![image](https://user-images.githubusercontent.com/49145827/196175735-8dddc45a-0b17-4df7-9d32-5da9c2c767bc.png)

### TLDR
the comments below are about the minimized version ^ the one above is the one that should be implemented, no need to check the comments below.

**Comment by KhizerRehan** [comment]
Any thoughts on this collapse section for Admin Panel sidebar?
https://user-images.githubusercontent.com/17727069/190964164-f3e790b6-7190-4480-a2a2-43213a95b236.mp4

cc: @cschieder

**Comment by KhizerRehan** [comment]
Just to add i think following ticker issue is same what i have discussed above:
https://github.com/kubermatic/dashboard/issues/3468

cc: @cschieder @Waseem826

**Comment by cschieder** [comment]
Good point, some drafts - 1. is my favorite, thoughts? 

1) Shortening and truncating
![image](https://user-images.githubusercontent.com/49145827/190986502-81545540-cf3f-4066-9e94-aaf62d7b9717.png)
---
2) Dots (works without the collapse arrow too, the icon will simply have collapse behavior) 
![image](https://user-images.githubusercontent.com/49145827/190986570-ecdba7c2-5604-491b-9ff1-0a27399fcd38.png)
---
3) Abbreviations/shorthands
![image](https://user-images.githubusercontent.com/49145827/190987535-ce17b60d-6be9-4b01-ba24-e4aed361377a.png)

**Comment by KhizerRehan** [comment]
I have a suggestion can we add a popup and show complete name instead of `Shortening and truncating` wdyt?

see this
![mobile](https://user-images.githubusercontent.com/17727069/191007392-e94ebb08-d7ce-47bb-b06e-190dae63b714.png)

We can do for this Admin Panel.
![ui-mob](https://user-images.githubusercontent.com/17727069/191007398-a00cfcbd-ec2b-4a36-b544-2bdf068f15f6.png)

**Comment by cschieder** [comment]
That works for me too

**Comment by cschieder** [comment]
Drafts for popup
![image](https://user-images.githubusercontent.com/49145827/192268226-e3dc2f08-1e4c-45e9-9cf5-e8afcfd7ecc6.png)
@KhizerRehan

**Comment by KhizerRehan** [comment]
@cschieder can you share Dark version too

**Comment by cschieder** [comment]
The thought for 1 was that it pushes the other icons down but that would probably be a hassle to do any be a bit odd for usability so lets scratch that 

Dark Theme
![image](https://user-images.githubusercontent.com/49145827/192489407-c95cc776-0475-4d84-9fbf-bc57bd0dbb41.png)

**Comment by cschieder** [comment]
One more variant
![image](https://user-images.githubusercontent.com/49145827/195559084-c55fd76e-149a-49de-b782-8052b7f20fde.png)
